utils/verdaccio-starter/src/index.ts

```diff
@@ -0,0 +1,78 @@
+// eslint-disable-next-line node/no-unpublished-import
+import {ConfigRuntime} from '@verdaccio/types';
```

Comment:
```suggestion
import type {ConfigRuntime} from '@verdaccio/types';
```

Should fix the node/no-unpublished-import iirc